docs/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md

```diff
@@ -78,25 +78,13 @@ Synchronize files between Web Modeler and version control systems (VCS) and vice
 
 For automatic file synchronization, consider maintaining a secondary system of record for mapping Web Modeler projects to VCS repositories. This system also monitors the project-to-repository mapping and update timestamps.
 
-To listen to changes in Web Modeler, you currently need to implement a polling approach that compares the update dates with the last sync dates recorded. Use the `POST /api/v1/files/search` [endpoint](https://modeler.cloud.camunda.io/swagger-ui/index.html#/Files/searchFiles) with this payload to identify recently updated files:
+To listen to changes in Web Modeler, you currently need to implement a polling approach that compares the update dates with the last sync dates recorded. Use the `POST /api/v1/files/search` [endpoint](https://modeler.cloud.camunda.io/swagger-ui/index.html#/Files/searchFiles) with the following payload to identify files updated after the last sync date:
 
 ```json title="POST /api/v1/files/search"
 {
   "filter": {
-    "projectId": "<PROJECT TO SYNC>",
-    "updated": "<LAST SYNC DATE>"
-  },
-  "page": 0,
-  "size": 50
-}
-```
-
-For real-time synchronization, employ a polling approach comparing update dates with last sync dates. Use the `POST /api/v1/files/search` [endpoint](https://modeler.cloud.camunda.io/swagger-ui/index.html#/Files/searchFiles) with the following payload to discover recently updated files, and compare the `updated` date with your last sync date:
-
-```json title="POST /api/v1/files/search"
-{
-  "filter": {
-    "projectId": "<PROJECT TO SYNC>"
+    "projectId": "{PROJECT TO SYNC}",
+    "updated": ">{LAST SYNC DATE}"
```

Comment:
@christinaausley This snippet now uses `{` (resp `}`) for indicating a variable contrary to line 179 where angle brackets are used (`"<PROJECT ID>"`). On the other hand, I'm not too happy with using curly brackets as they are already by indicating JSON objects. Do you have a good idea on how to unify this within this page?

Reply:
This is such a good question. Is there a reason you dislike the carrots `<`? Alternatively, we could use parentheses or italics.

Reply:
I see -- are you opposed to italics?

Reply:
Not at all 🙃